types/react/index.d.ts

```diff
@@ -1697,13 +1697,15 @@ declare namespace React {
     }
 
     interface TdHTMLAttributes<T> extends HTMLAttributes<T> {
+        align?: string;
```

Comment:
`align?: "left" | "center" | "right" | "justify" | "char";`
would be nice!

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/td#attr-align
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/th#attr-align

Reply:
Sure!